e2e_test.go

```diff
@@ -66,14 +72,17 @@ func TestPgxConnect(t *testing.T) {
 	if err != nil {
 		t.Fatalf("failed to init Dialer: %v", err)
 	}
+	// Close the dialer when you're done with the database connection
+	defer d.Close()
 
-	dsn := fmt.Sprintf("user=%s password=%s dbname=%s sslmode=disable", alloydbUser, alloydbPass, alloydbDB)
+	dsn := fmt.Sprintf("user=%s password=%s dbname=%s sslmode=disable",
```

Comment:
should we have a comment here mentioning why we are disabling `sslmode`. As a new user worried about security this might worry me seeing this...

Reply:
Good point. Done.